pkg/pillar/hypervisor/kvm.go

```diff
@@ -350,10 +350,11 @@ func newKvm() Hypervisor {
 		}
 	case "amd64":
 		return kvmContext{
+			//nolint:godox // FIXME: Removing "-overcommit", "mem-lock=on", "-overcommit" for now, revisit it later as part of resource partitioning
```

Comment:
What are these //nolint:godox tags about? Does golangcilint not like your comment?

Reply:
It does not like any FIXME or XXX in the comment, and that comes under "godox" category of golangci-lint checks

Reply:
Hm. We should probably look into that in that thread on golangcilint @eriknordmark  -- but we can leave it here for now